Cargo.toml

```
@@ -7,6 +7,7 @@ description = "Pyrsia Node and CLI"
 authors = ["pyrsiaoss <pyrsiaopensource@gmail.com>"]
 license = "Apache-2"
 repository = "https://github.com/pyrsia/pyrsia"
+rust = "1.65"
```

Regarding our recent problems with Clippy, we should downgrade rust to 1.64 until 1.67 is stable (currently nightly)
```suggestion
rust = "1.64"
```

---

change to 1.64